helm_deploy/values-dev.yaml

```diff
@@ -74,6 +74,7 @@ generic-service:
     PROFILE_ADD_APPOINTMENT_ENABLED: "true"
     USE_OF_FORCE_DISABLED_PRISONS: "CKI,FYI,WNI,WYI"
     COURT_CASES_SUMMARY_ENABLED: "true"
+    NEW_COURSE_AND_QUALIFICATION_HISTORY_ENABLED: "true"
```

Comment:
The feature toggle for this new Courses And Qualifications card was added a little while back; now we are enabling it in `dev`